bootstrap-themes/src/main/java/de/agilecoders/wicket/themes/markup/html/italia/ItaliaTheme.java

```diff
@@ -0,0 +1,64 @@
+package de.agilecoders.wicket.themes.markup.html.italia;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.apache.wicket.Application;
+import org.apache.wicket.markup.head.CssHeaderItem;
+import org.apache.wicket.markup.head.HeaderItem;
+import org.apache.wicket.markup.head.JavaScriptHeaderItem;
+import org.apache.wicket.markup.head.PriorityHeaderItem;
+
+import de.agilecoders.wicket.core.settings.Theme;
+import de.agilecoders.wicket.webjars.request.resource.WebjarsCssResourceReference;
+import de.agilecoders.wicket.webjars.request.resource.WebjarsJavaScriptResourceReference;
+
+/**
+ * A {@link de.agilecoders.wicket.core.settings.ITheme theme} for Bootstrap
+ * provided by <a href="https://developers.italia.it/">developers.italia.it</a>.
+ * 
+ * <h2>Usage</h2>
+ * <pre>settings.setThemeProvider(new SingleThemeProvider(new ItaliaTheme()));</pre>
+ *
+ * @author Erik Geletti
+ */
+public class ItaliaTheme extends Theme {
+
+	/**
+     * Construct.
+     */
+    private ItaliaTheme(final String name) {
+    	super(name);
+    }
+    
+    /**
+     * Construct.
+     */
+    public ItaliaTheme() {
+        this("italia");
+    }
+    
+    @Override
+    public List<HeaderItem> getDependencies() {
+		List<HeaderItem> references = new ArrayList<HeaderItem>();
+		references.add(
+				CssHeaderItem.forReference(new WebjarsCssResourceReference("splide/current/css/splide-core.css")));
+		references.add(CssHeaderItem
+				.forReference(
+						new WebjarsCssResourceReference("bootstrap-italia/current/css/bootstrap-italia.css"))
+				.setId(BOOTSTRAP_THEME_MARKUP_ID));
+		references.add(new PriorityHeaderItem(JavaScriptHeaderItem
+				.forReference(Application.get().getJavaScriptLibrarySettings().getJQueryReference())));
+		references.add(JavaScriptHeaderItem.forReference(
+				new WebjarsJavaScriptResourceReference("popper.js/current/umd/popper.js")));
+		references.add(JavaScriptHeaderItem.forReference(
+				new WebjarsJavaScriptResourceReference("splide/current/js/splide.js")));
+		references.add(JavaScriptHeaderItem.forScript("window.__PUBLIC_PATH__ = '/wicket/resource/de.agilecoders.wicket.webjars.request.resource.IWebjarsResourceReference/webjars/bootstrap-italia/current/fonts'", "bootstrap-italia-fonts"));
```

Comment:
What is this ?
The hardcoded path looks problematic. I am not sure it will work for every application. It may lead to broken url when the application is deployed at non-ROOT context.

Reply:
I'm looking for a solution. I have to set in JS the variable `window.__PUBLIC_PATH__` to the path of bootstrap-italia fonts `bootstrap-italia/current/fonts`.
I'm looking how get a URL of a WebjarResourceReference.

Reply:
You may use `RequestCycle#urlFor()` methods.

Reply:
This could be an idea:
```suggestion
		references.add(JavaScriptHeaderItem.forScript("window.__PUBLIC_PATH__ = '" + RequestCycle.get().urlFor(new PackageResourceReference(WebjarsCssResourceReference.class, "/webjars/bootstrap-italia/current/fonts"), null) + "'", "bootstrap-italia-fonts"));
```

Reply:
To make this a bit cleaner I made this PR: l0rdn1kk0n/wicket-webjars#68